module4/frontend/dist/related-bundle.js

```diff
@@ -0,0 +1,578 @@
+/*
```

Comment:
i think you accidentally included this one

Reply:
yes, ill add that to gitignore